main/static/main/CSS/style.css

```diff
@@ -0,0 +1,122 @@
+{% extends "main/base.html" %}
+
+.nav_area .main_menu .navbar .navbar-band{
+    padding: 0 2rem 0 5rem;
+}
+
+.nav_area .main_menu .navbar{
+    padding: .75rem 8rem;
+    background-color: #F5F5F5;
+}
+.nav_area .main_menu .nav-item .nav-link{
+    font-family: Roboto,cursive;
+    font-weight: 500;
+    text-transform: uppercase;
+    padding: 1rem;
+    color: black;
+}
+
+.nav_area .main_menu .navbar-nav active a{
+    background-color: #65BF02;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-clip: text;
+}
+.nav_area .main_menu .navbar-nav a:hover{
+    background-color: #65BF02;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-clip: text;
+}
+/*header Area or Nav area has Finished*/
+
+.site-main .site-banner .site-title h3{
+    padding-top: 40%;
+    padding-left: 10%;
+    font-size: 2.5rem;
+    font-weight: bold;
+    color: #9F9F9F;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.site-main .site-banner .site-title h2{
+    padding-top: 5%;
+    padding-bottom: 10%;
+    padding-left: 10%;
+    font-size: 2.5rem;
+    font-weight: bold;
+    color: #9F9F9F;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.site-main .site-banner .site-title span{
+    color: black;
+    font-family: Ubuntu, sans-serif;
+}
+
+/* From Here I write code for box*/
+
+.site-main .site-banner .row .site-search-btn
+{
+    padding-top: 12%;
+}
+/* ----------------Result HTML-------------*/
+.container .box-search-in-result .search {
+	padding: 0 32px;
+	font-size: 16px;
+	width: 60%;
+	max-width: 700px;
+	height: 54px;
+	border: 1px solid darken(white, 30%);
+	border-radius: 20px 0 0 20px;
+}
+{% load static %}
+
+.container .box-search-in-result .submit {
+	cursor: pointer;
+	border: none;
+	background:  url('{{ STATIC_URL }}main/images/Search-icon.png') no-repeat center center, #1E1E20;
+	background-size: 25px 25px;
+	border-radius: 0 30px 30px 0;
+	padding: 15px 25px;
+	display: inline-block;
+    width: 20%;
+}
+.container .row-in-result
+{
+    padding-top: 1.5%;
+    padding-left:15%;
+}
+/*---------------------------Result HTML END------------------------------*/
+.box-search{
+    padding-left: 10%;
+    padding-top: 22%;
+    transition: 0.5s;
+}
+.search {
+	padding: 0 32px;
+	font-size: 16px;
+	width: 60%;
+	max-width: 400px;
+	height: 54px;
+	border: 1px solid darken(white, 30%);
+	border-radius: 20px 0 0 20px;
+}
+
+.submit {
+	cursor: pointer;
+	border: none;
+	background:  url('{{ STATIC_URL }}main/images/Search-icon.png') no-repeat center center, #1E1E20;
```

Comment:
this line should be `background:  url("../images/Search-icon.png") no-repeat center center, #1E1E20;`